packages/layouts-css/src/page.scss

```diff
@@ -94,5 +96,71 @@
         justify-content: flex-end;
       }
     }
+
+    &-centered {
```

Comment:
Move back to original place (to avoid unnecessary changes)

Reply:
I have moved this back